crates/db_queries/src/source/comment.rs

```diff
@@ -135,7 +135,9 @@ impl Comment_ for Comment {
   }
 }
 
-impl Crud<CommentForm, CommentId> for Comment {
+impl Crud for Comment {
+  type Form = CommentForm;
+  type IdType = CommentId;
   fn read(conn: &PgConnection, comment_id: CommentId) -> Result<Self, Error> {
```

Comment:
Its not required, but its generally a good practice, and we should do this, is to use these trait types instead of the specific named ones. below. IE `fn read(conn: &PgConnection, comment_id: Self::IdType)`

Reply:
Not sure if thats better. It might make things easier if we change the type a lot (cause we only need to change it in one place), but it might also be harder to understand the code if you have to look it up all the time. And either way the compiler will complain if its wrong.